variant_annotation/wdl/scrape_annot_v4.wdl

```diff
@@ -339,25 +352,25 @@ task add_gnomad {
             for line in f:
                 oline = line.rstrip("\r\n")
                 s = oline.split('\t')
-                chrom = int(s[h_idx['chr']])
+                chrom = format_chrom(s[h_idx['chr']])
                 pos = int(s[h_idx['pos']])
                 ref = s[h_idx['ref']]
                 alt = s[h_idx['alt']]
                 #check if gen and exo vars have different c:p than variant. If yes, empty them.
                 if gen_vars:
-                    if int(gen_vars[0][gen_ref.h_idx['#CHROM']]) != chrom or int(gen_vars[0][gen_ref.h_idx['POS']]) != pos:
+                    if format_chrom(gen_vars[0][gen_ref.h_idx['#CHROM']]) != chrom or int(gen_vars[0][gen_ref.h_idx['POS']]) != pos:
                         gen_vars = []
 
                 if exo_vars:
-                    if int(exo_vars[0][exo_ref.h_idx['#CHROM']]) != chrom or int(exo_vars[0][exo_ref.h_idx['POS']]) != pos:
+                    if format_chrom(exo_vars[0][exo_ref.h_idx['#CHROM']]) != chrom or int(exo_vars[0][exo_ref.h_idx['POS']]) != pos:
                         exo_vars = []
 
                 #genome
                 while gen_ref.has_lines and gen_ref.chrom < chrom or ( gen_ref.chrom == chrom and gen_ref.pos < pos):
                     gen_ref.line = gen_ref.fp.readline().rstrip('\r\n').split('\t')
```

Comment:
rather check if the line is empty string indicating eof than rely on number format errors to declare not having any more lines. Throw error if there is non numeric value in pos